README.md

```diff
@@ -16,6 +16,39 @@ In particular:
 
 See the documentation for the specific tests for more information and references.
```

Comment:
While you're at it, could you fix this sentence too? I think "for the specific tests" should be removed.

Reply:
And perhaps, given the info you added, this sentence can be removed completely.